src/core/agent/modules/signify/signifyApi.types.ts

```diff
@@ -106,6 +106,7 @@ export enum NotificationRoute {
   Credential = "/exn/ipex/grant",
   MultiSigIcp = "/multisig/icp",
   MultiSigRot = "/multisig/rot",
+  GrantRequest = "/tunnel/wallet/request",
```

Comment:
Lets call it `TunnelRequest` - me using the word "grant" was probably a mistake since there is also grant in IPEX.